### PR TITLE
Start menu

### PR DESCRIPTION
**Pull request by Nolfews**
fix leave button for exit game and not when we press escape

### CODE REVIEW

File: src/Modules/PlayerModule.hpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#include "RotatableModule.hpp"
+#include "TimeModule.hpp"
+#include "PositionModule.hpp"
+#include "HitboxModule.hpp"
+
+class PlayerModule : public Module {
+    public:
+        PlayerModule(GameObject *gameObject);
+        ~PlayerModule() = default;
+    private:
```
Comment (Thomaltarix) [inline src/Modules/PlayerModule.hpp]: The Player has to declare the update method to success the compilation

File: src/Game/Game.cpp [hunk 2]
```diff
@@ -88,6 +96,44 @@ void Game::handleMouseEvents()
...
+            if (button->getType() == button->DOOR) {
+                rect = sf::IntRect(144,0,72,66);
+                button->getSprite()->setTextureRect(rect);
+                rect = sf::IntRect(216,0,72,66);
+                button->getSprite()->setTextureRect(rect);
+            } else {
+                rect = sf::IntRect(0,266,500,133);
+                button->getSprite()->setTextureRect(rect);
+            }
+            if (button->getType() == 3) {
```
Comment (Thomaltarix) [inline src/Game/Game.cpp]: '3' Must be an Enum

**Review by Thomaltarix — CHANGES_REQUESTED** [review]
Some things to change

**Review by eliotttesnier — APPROVED** [review]
ouais

**Pull request merged**